src/elements/content-sidebar/activity-feed/task-new/AssigneeList.js

```diff
@@ -0,0 +1,162 @@
+// @flow strict
+import * as React from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import type { InjectIntlProvidedProps } from 'react-intl';
+import uniqueId from 'lodash/uniqueId';
+import PlainButton from '../../../../components/plain-button';
+import ReadableTime from '../../../../components/time/ReadableTime';
+import messages from '../../../common/messages';
+import AvatarGroupAvatar from './AvatarGroupAvatar';
+import { TASK_NEW_APPROVED, TASK_NEW_REJECTED, TASK_NEW_COMPLETED, TASK_NEW_NOT_STARTED } from '../../../../constants';
+
+import './AssigneeList.scss';
+
+const MIN_ASSIGNEES_SHOWN = 3;
+const TASKS_PAGE_SIZE = 20; // service does not return the page size to the client at the moment
+
+type Props = {|
+    getAvatarUrl: GetAvatarUrlCallback,
+    minAssignees: number,
```

Comment:
something like initialAssignee count might help enforce the point across that this is expandable.

Reply:
`minAssignees` isn't quite accurate since you can have less than 3 initially